content/gocourse-2019-05-melbourne.slide

```diff
@@ -602,7 +594,18 @@ Before adding any commits to `feature` locally run:
 	language:bash
 	git pull origin feature
 
+* Lab PR approval flow - Programmer
+
+- Tag with the appropriate lab tag (eg: `lab1`)
+- When ready, tag with `Ready` `for` `review`
+- Wait for PR to be tagged `Approved` `by` `colleague` or `Changed` `requested`
+- If approved by colleague, wait for a CODEOWNER to approve and tag with `Merge` `me`
+- Merge
+
+* Lab PR approval flow - Reviewer
 
+- Remove `ready for review` and add `under review`
```

Comment:
\`Ready\` \`for\` \`review\` and add \`Under\` \`review\`

(capitalisation and spacing)